File: 240603/문자열의 특정 위치 찾기 2/find-specific-location-in-spring-2.py
# arr = ["apple", "banana", "grape", "blueberry", "orange"]

# w = input()

# a = arr[0].count(w)
# b = arr[1].count(w)
# c = arr[2].count(w)
# d = arr[3].count(w)
# e = arr[4].count(w)

# print(a > 0)

# 5개의 문자열 초기화
words = ["apple", "banana", "grape", "blueberry", "orange"]

# 첫 번째 줄 입력 받기 (영문자 하나)
char = input("")

# 조건을 만족하는 문자열들을 저장할 리스트
matching_words = []

# 조건에 맞는지 확인하고, 맞으면 리스트에 추가
for word in words:
    if len(word) >= 3 and (word[2] == char or (len(word) >= 4 and word[3] == char)):
        matching_words.append(word)

# 결과 출력
for word in matching_words:
    print(word)

# 조건을 만족하는 문자열의 개수 출력
print(len(matching_words))

File: 240822/문자열의 특정 위치 찾기 2/find-specific-location-in-spring-2.js
// 변수 선언 및 입력
const fs = require("fs");

// 영문자를 입력받습니다.
let a = fs.readFileSync(0).toString().trim();

// 문자열 리스트를 정의합니다.
let string = ["apple", "banana", "grape", "blueberry", "orange"]

// 조건을 만족하는 문자열을 출력하고 조건을 만족하는 문자열의 개수를 셉니다.
let cnt = 0;
for (let i = 0; i < 5; i++) {
    if (string[i][2] === a || string[i][3] === a) {
        console.log(string[i]);
        cnt++;
    }
}

// 조건을 만족하는 문자열의 개수를 출력합니다.
console.log(cnt);